docs/src/pages/components/tree-view/GmailTreeView.js

```diff
@@ -34,6 +34,10 @@ const useTreeItemStyles = makeStyles((theme) => ({
       backgroundColor: `var(--tree-view-bg-color, ${theme.palette.action.selected})`,
       color: 'var(--tree-view-color)',
     },
+    '& $label': {
```

Comment:
Should we use the global class here? For example:
```suggestion
    '& .MuiTreeItem-label': {
```

Reply:
Or even better: 
```suggestion
    '& .${treeItemClasses.label}': {
```

Reply:
@mnajdova similar to @oliviertassinari suggestion, but there are many places still using `$` in this file so I think it might be better to update the whole demo in another PR. is that okay?

Reply:
ok, got it, sorry missed previous question I guess :)